lib/types.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+
+import React from 'react';
+
+export type ItemType = {
```

Comment:
Can we call these `TextItemType` and `TextItemsType` instead? Makes it clear this is for text only, that it goes together with the renderText function.